doc/report.md

```diff
@@ -5,14 +5,79 @@
 [Spadoni Marta](mailto:alessia.rocco@studio.unibo.it),
 [Rocco Alessia](mailto:marta.spadoni2@studio.unibo.it)
 
+## Descrizione
+`pps-bunny` è un simulatore per osservare l'evoluzione naturale di una famiglia di conigli nel corso delle generazioni.
+Per far interagire l'utente con la simulazione è prevista una interfaccia grafica game-like che permetta di inserire 
+delle mutazioni sia dei conigli che dell'ambiente circostante. 
+
+La proposta si ispira al seguente gioco: https://phet.colorado.edu/en/simulation/natural-selection.
+
 ## Processo di sviluppo adottato
+//Parlare di scrum, come lo abbiamo adottato e come abbiamo gestito il lavoro (Sprint di 2 settimane) e parlare del 
+risultato prodotto da ogni sprint:
+1. Sprint 1: abbiamo prodotto una modellazione concettuale che copre il modello del problema
+2. Sprint 2: abbiamo prodotto una prima versione dell'applicazione in cui è possibile osservare la riproduzione dei 
+conigli 
+3. Sprint 3: abbiamo integrato alla prima versione dell'app la possibilità di introdurre mutazioni genetiche nei conigli
+4. Sprint 4:
+
+### Modalità di revisione dei task
+Durante lo Sprint Planning viene assegnato a ogni membro l'insieme di task che esso ha il compito di portare a termine entro lo sprint successivo.
+A ciascuno dei task che sono stati pianificati viene quindi definita una Issue su GitHub con la persona incaricata e 
```

Comment:
Per ciascun task pianificato vengono definite una Issue su GitHub e una scheda Trello nella sezione "To Do" entrambe assegnate alla persona incaricata (di implementare il task)